components/match2/wikis/sideswipe/match_group_input_custom.lua

```diff
@@ -0,0 +1,578 @@
+		or {
+			date = MatchGroupInput.getInexactDate(globalVars:get('tournament_enddate')),
+			dateexact = false,
+		}
+end
+
+function matchFunctions.getTournamentVars(match)
+	match.mode = Logic.emptyOr(match.mode, Variables.varDefault('tournament_mode', '3v3'))
+	match.type = Logic.emptyOr(match.type, Variables.varDefault('tournament_type'))
+	match.tournament = Logic.emptyOr(match.tournament, Variables.varDefault('tournament_name'))
+	match.tickername = Logic.emptyOr(match.tickername, Variables.varDefault('tournament_ticker_name'))
+	match.shortname = Logic.emptyOr(match.shortname, Variables.varDefault('tournament_shortname'))
+	match.series = Logic.emptyOr(match.series, Variables.varDefault('tournament_series'))
+	match.icon = Logic.emptyOr(match.icon, Variables.varDefault('tournament_icon'))
+	match.icondark = Logic.emptyOr(match.iconDark, Variables.varDefault('tournament_icon_dark'))
+	match.liquipediatier = Logic.emptyOr(
+		match.liquipediatier,
+		Variables.varDefault('tournament_lptier'),
+		Variables.varDefault('tournament_tier')
+	)
+	match.liquipediatiertype = Logic.emptyOr(
+		match.liquipediatiertype,
+		Variables.varDefault('tournament_tiertype')
+	)
+	return match
+end
+
+function matchFunctions.getVodStuff(match)
+	match.stream = match.stream or {}
+	match.stream = {
+		stream = Logic.emptyOr(match.stream.stream, Variables.varDefault('stream')),
+		twitch = Logic.emptyOr(match.stream.twitch or match.twitch, Variables.varDefault('twitch')),
+		twitch2 = Logic.emptyOr(match.stream.twitch2 or match.twitch2, Variables.varDefault('twitch2')),
+		afreeca = Logic.emptyOr(match.stream.afreeca or match.afreeca, Variables.varDefault('afreeca')),
+		afreecatv = Logic.emptyOr(match.stream.afreecatv or match.afreecatv, Variables.varDefault('afreecatv')),
+		dailymotion = Logic.emptyOr(match.stream.dailymotion or match.dailymotion, Variables.varDefault('dailymotion')),
+		douyu = Logic.emptyOr(match.stream.douyu or match.douyu, Variables.varDefault('douyu')),
+		smashcast = Logic.emptyOr(match.stream.smashcast or match.smashcast, Variables.varDefault('smashcast')),
+		youtube = Logic.emptyOr(match.stream.youtube or match.youtube, Variables.varDefault('youtube'))
+	}
+	match.vod = Logic.emptyOr(match.vod, Variables.varDefault('vod'))
+
+	-- apply vodgames
+	for index = 1, MAX_NUM_VODGAMES do
+		local vodgame = match['vodgame' .. index]
+		if not Logic.isEmpty(vodgame) then
+			local map = match['map' .. index] or {}
+			map.vod = map.vod or vodgame
+			match['map' .. index] = map
+		end
+	end
+	return match
+end
+
+function matchFunctions.getExtraData(match)
+	local opponent1 = match.opponent1 or {}
+	local opponent2 = match.opponent2 or {}
+	match.extradata = {
+		matchsection = Variables.varDefault('matchsection'),
+		team1icon = getIconName(opponent1.template or ''),
+		team2icon = getIconName(opponent2.template or ''),
+		lastgame = Variables.varDefault('last_game'),
+		comment = match.comment,
+		octane = match.octane,
```

Comment:
Octane doesn't support ss based on discord conv
Also shouldn't it be in match.links if it were?